pysilcam/silcamgui/liveviewer.py

```diff
@@ -21,9 +21,11 @@
 import pandas as pd
 import time
 from pysilcam.__main__ import *
-import pygame
 from skimage.io import imsave as imwrite
 
+os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
+import pygame
```

Comment:
E402: module level import not at top of file  <!-- Automatically posted by Lintly -->